### DISCUSSION TITLE
What happens when an Index Optimize or Integrity Check fail for one DB?

### DISCUSSION THREAD

**Discussion by Lumute** [olahallengren/sql-server-maintenance-solution#515] [Questions & Answers]
Hi,

I implemented this solution and so far index optimization and integrity check jobs are working beautifully for all my DBs.

That said, I am wondering, what happens when one of those tasks fail for one of the DBs, will the whole Job report failure? and if so, what happens with the remaining of the DBs, does the job continues for the DBs that are still pending or will it just stop processing on the DB it failed based on the order specified?

Thanks!

**Comment by olahallengren** [comment]
It will continue processing the remaining databases. The job will report failure in the end.

**Reply by naeembashir123** [reply]
I get the job reporting a failure on the indexing however it is giving an errors in the commandlog table.  the job reports the following:

(Message 50000)  Date and time: 2022-0...  The step failed.

**Comment by jacobgexigo** [comment]
If the failure is from deadlock or lock timeouts, you can also use the LockMessageSeverity parameter to allow it to be an informational message instead of a failure messge.

![image](https://user-images.githubusercontent.com/29784949/159707111-0b4cc2db-2451-43cd-bd72-03c837520569.png)
